code/game/gamemodes/antag_spawner.dm

```diff
@@ -119,40 +122,81 @@
 	target.equip_to_slot_or_del(new /obj/item/weapon/storage/box(target), slot_in_backpack)
 	target.equip_to_slot_or_del(new /obj/item/weapon/teleportation_scroll/apprentice(target), slot_r_store)
 
-/obj/item/weapon/antag_spawner/borg_tele
-	name = "syndicate cyborg teleporter"
-	desc = "A single-use teleporter designed to deploy a single Syndicate cyborg onto the field."
+
+
+
+
+
+///////////BORGS AND OPERATIVES
+
+
+/obj/item/weapon/antag_spawner/nuke_ops

```

Comment:
This works I guess, not exactly what I had in mind (I expected two children of this, one for nukies and one for borgies), but this is acceptable.